ambiata-mafia.cabal

```diff
@@ -22,6 +22,7 @@ library
                     , attoparsec                      >= 0.12       && < 0.14
                     , base16-bytestring               == 0.1.*
                     , bytestring                      == 0.10.*
+                    , Cabal                           >= 1.22       && < 1.25
```

Comment:
I just don't want to depend on the `Cabal` library if I can avoid it, it opens the door to mafia needing to be built again the same `Cabal` library as the `cabal-install` executable. Currently we can use the same mafia executable with different `cabal-install` versions and manage the differences dynamically.